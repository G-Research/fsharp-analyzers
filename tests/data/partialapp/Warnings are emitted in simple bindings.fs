module M

let myFunc x y = x + y

let partapp2 = myFunc 4 // should warn
