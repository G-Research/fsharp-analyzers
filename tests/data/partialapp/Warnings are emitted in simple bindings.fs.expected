GRA-PARTAPP-001 | Warning | (5,15 - 5,21) | partial application should not be used: myFunc at (5,15--5,21)
